---
title: "Resume"
date: 2025-08-10T12:00:00+02:00
draft: false
---

<a href="/VivekAppaduraiCV.pdf" target="_blank" rel="noopener">Download CV</a>

## Professional Summary

I'm a bioinformatics professional with over 12 years of experience in pharmaceutical research, clinical science and academic settings, specializing in the development, modernization, and support of scientific software solutions and informatics processes. I possess proven expertise in Python, SQL, automating workflows, using high performance, cloud computing platforms, and contributing to AI/ML-enabled drug discovery initiatives. I'm adept at bridging scientific research needs with technical execution, driving data automation, and optimizing underlying databases. In addition, I have demonstrated strong problem-solving, collaboration and communication skills, learning agility, along with a commitment to delivering user-friendly, research-centric solutions within dynamic software ecosystems.

## Skills

*   **Programming & Scripting:** Python, R, Perl, SQL, Bash, Git, RESTful APIs, Docker.
*   **Software Development & DevOps:** Microservice Architecture, Azure pipelines for CI/CD, Agile Methodologies, Software System Maintenance & Enhancement, Technical Documentation.
*   **AI/ML:** Hypothesis Testing, Regression Modeling, Classification, Clustering, Imputation, Dimensionality Reduction, Conceptual & Logical Data Modeling, Ontologies.
*   **Data Management:** Relational Databases (Optimization & Restructuring), Data Registration, Metadata, Master and Reference Data Management, Data Lineage, Data Governance, Information Security, Data Complexity Management.
*   **Cloud & HPC:** AWS (S3, EC2), Sun Grid Engine, MOAB, LSF.
*   **Data Visualization:** Streamlit, Shiny, Dash, Tibco Spotfire.
*   **Domain Expertise:** Bioinformatics, RDKit, Omics Data Integration, Pharmaceutical Research & Development workflows.
*   **Languages:** English (Native speaker), Danish (Intermediate, passed PD3)

## Experience

### Senior Technical Data Steward, Novo Nordisk
Maaloev, Denmark | Sep 2023 - Present
At Novo Nordisk, I led the development of a microservice application to improve metadata capture for AI/ML-driven drug discovery. I designed and implemented data models for the compound database, authored architectural decision records for modernization efforts, and established data governance standards. I also provided hands-on support in bioinformatics, data visualization, and machine learning, and mentored junior colleagues.

### Postdoctoral Researcher, Institute of Biological Psychiatry
Roskilde, Denmark | Apr 2021 - Sep 2023
As a Postdoctoral Researcher, I designed and built a comprehensive bioinformatics data infrastructure for the iPSYCH consortium. I led a research study using machine learning on genetic and transcriptomic data to identify novel genetic features for psychiatric disease research. I also collaborated with clinical researchers to identify and address critical research challenges.

### Doctoral Intern in Bioinformatics, Lundbeck A/S
Valby, DK | Aug 2020 - Oct 2020
During my internship at Lundbeck, I contributed to the target discovery phase by providing real-world evidence. I calculated the prevalence of loss-of-function mutations in potential targets for severe migraine, gaining direct experience in biopharma scientific software platforms.

### PhD Candidate, Institute of Biological Psychiatry
Roskilde, DK | Jan 2018 - Apr 2021
As a PhD candidate, I led bioinformatics analyses for several research studies. I applied regression modeling and causal inference methods, and empirically estimated the impact of missing data imputation algorithms on genetic risk scores, demonstrating strong analytical and problem-solving skills.

### Lead Bioinformatician, Institute of Biological Psychiatry
Roskilde, DK | July 2016 - Dec 2018
In my role as Lead Bioinformatician, I led efforts to improve the quality of large exome sequencing datasets. I addressed data degradation issues and contributed to highly-cited studies, demonstrating my expertise in data analysis and quality improvement.

### Bioinformatician, Human Genome Sequencing Center
Houston, TX USA | Mar 2012 – Jun 2016
At the Human Genome Sequencing Center, I developed software and automated workflows to identify candidate causal mutations for pediatric-onset mitochondrial diseases. I integrated evidence from open-source clinical variant repositories and applied machine learning-based variant effect prediction models, leading to a significant increase in the resolution of unsolved cases.

### Scientific Programmer, The McDowell Genome Institute
St. Louis, USA | Jan 2010 - Dec 2011
As a Scientific Programmer at The McDowell Genome Institute, I developed software to improve the scaffolding quality of de novo genome assemblies. I delivered these assemblies to the UCSC genome browser and contributed to open-source scientific data repositories.

## Education

### University of Copenhagen
PhD in Biostatistics and Bioinformatics | Apr 2021

### Missouri University of Science and Technology
MS in Electrical and Computer Engineering | Dec 2009

### Anna University
BE in Electronics Engineering | May 2006

## Awards

### Lundbeck Foundation - Postdoctoral Fellowship
Apr 2022

## Extracurricular

I'm an avid runner (PRs 10k: 46.01, 5k: 22:30) and I enjoy reading neo-western, classical American, Russian literature when I'm not spending time with my soon to be one-year old daughter. Prior to becoming a dad, I volunteered with the coding pirates branch in Østerbro, teaching programming skills in Python and Javascript to teenagers.